indico/web/templates/indico_base.html

```diff
@@ -41,7 +41,7 @@
 
     {% block late_js %}
         <script>
-            moment.locale([{{ session.moment_lang|tojson }}, 'en']);
```

Comment:
Don't we want to keep this? If a locale doesn't exist in moment we want to fall back to `en` and not whatever else it picks by default...

Reply:
`session.moment_lang` already defaults to `en` so this shouldn't be needed unless a future version of moment removes/renames some locales. But for safety we could keep it anyway